ui/src/components/Equipment/PlateManipulatorMaintenance.js

```diff
@@ -0,0 +1,210 @@
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+import { Card, Button, ButtonGroup, InputGroup, Form } from 'react-bootstrap';
+// import Range from '../Range/EditableRange';
+import './Equipment.css';
```

Comment:
It would be nice to use css modules, see

    https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

Reply:
sure